apps/files/src/filters/TypeFilter.ts

```diff
@@ -142,7 +145,8 @@ class TypeFilter extends FileListFilter {
 	}
 
 	public setPreset(presets?: ITypePreset[]) {
-		this.currentPresets = presets
+		this.currentPresets = presets ?? []
+		this.currentInstance!.$props.preset = presets
```

Comment:
```suggestion
		this.currentInstance!.$props.preset = presets ?? []
```
The proptype is an array so we could do this and remove the `|| []` in FileListFilterType?